### DISCUSSION TITLE
Undefined class 'DatabaseAccessor'.

### DISCUSSION THREAD

**Discussion by dongri727** [serverpod/serverpod#2822] [Q&A] [answered]
After updating to Serverpod version 2.1.3, I encountered the issue shown in the attached image.

I have already run dart pub global activate serverpod_cli and aligned the version to 2.1.3 in the pubspec.yaml file.

Besides these two actions, is there anything else I need to do?
How can I address this error?

Here is my current environment setup:

flutter --version
Flutter 3.24.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 2663184aa7 (3 weeks ago) • 2024-09-11 16:27:48 -0500
Engine • revision 36335019a8
Tools • Dart 3.5.3 • DevTools 2.37.3

flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.24.3, on macOS 14.6.1 23G93 darwin-arm64, locale ja)
[✓] Android toolchain - develop for Android devices (Android SDK version 33.0.1)
[✓] Xcode - develop for iOS and macOS (Xcode 16.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.1)
[✓] VS Code (version 1.93.1)
[✓] Connected device (5 available)
[✓] Network resources

• No issues found!

<img width="1275" alt="スクリーンショット 2024-10-06 17 27 40" src="https://github.com/user-attachments/assets/74fb0b5a-bf7d-4120-bc7c-15b56a347851">

**Comment by hampuslavin** [comment, marked as answer]
Hi @dongri727!
That file has indeed been generated by the latest `serverpod_cli` version. What I'd like to double check is wether the dependency in your project has also been fetched. Can you double check that you have run `dart pub get` since you bumped the version? Could you also please paste your `pubspec.yaml` file for the server?

Thanks!

**Reply by dongri727** [reply]
Hi @hampuslavin !
I'm a real goofball; I freaked out when I saw Error and forgot all about the dart pub get. Thank you very much !
